cookbook/docs/tutorials.rst

```diff
@@ -39,4 +39,13 @@ documenting and contributing samples easy. If this is your first time running th
        :text: 🧪 Bioinformatics
        :classes: btn-block stretched-link
     ^^^^^^^^^^^^
-    Do computational biology with Flyte.
\ No newline at end of file
+    Perform computational biology with Flyte.
```

Comment:
@SmritiSatyanV let's create a new "Flytelab" section header below tutorials with a separate set of panels for flytelab projects. This is so we differentiate between "flytesnacks" examples and "flytelab" repo projects,